tests/ops/fixtures/saas/auth0_fixtures.py

```diff
@@ -0,0 +1,173 @@
+
+
+@pytest.fixture(scope="function")
+def auth0_connection_config(db: session, auth0_config, auth0_secrets) -> Generator:
+    fides_key = auth0_config["fides_key"]
+    connection_config = ConnectionConfig.create(
+        db=db,
+        data={
+            "key": fides_key,
+            "name": fides_key,
+            "connection_type": ConnectionType.saas,
+            "access": AccessLevel.write,
+            "secrets": auth0_secrets,
+            "saas_config": auth0_config,
+        },
+    )
+    yield connection_config
+    connection_config.delete(db)
+
+
+@pytest.fixture
+def auth0_dataset_config(
+    db: Session,
+    auth0_connection_config: ConnectionConfig,
+    auth0_dataset: Dict[str, Any],
+) -> Generator:
+    fides_key = auth0_dataset["fides_key"]
+    auth0_connection_config.name = fides_key
+    auth0_connection_config.key = fides_key
+    auth0_connection_config.save(db=db)
+    dataset = DatasetConfig.create(
+        db=db,
+        data={
+            "connection_config_id": auth0_connection_config.id,
+            "fides_key": fides_key,
+            "dataset": auth0_dataset,
+        },
+    )
+    yield dataset
+    dataset.delete(db=db)
+
+
+@pytest.fixture(scope="function")
+def auth0_erasure_data(
+    auth0_connection_config, auth0_erasure_identity_email, auth0_secrets
+) -> Generator:
+    """
+    Creates a dynamic test data record for erasure tests.
+    Yields user ID as this may be useful to have in test scenarios
+    """
+
+    base_url = f"https://{auth0_secrets['domain']}"
+    # Create user
+    body = {
+        "client_id": auth0_secrets["client_id"],
+        "email": auth0_erasure_identity_email,
+        "password": "p@sswordtest1",
+        "connection": "Username-Password-Authentication",
+        "username": "testaccount",
+        "given_name": "test",
+        "family_name": "account",
+        "name": "test account",
+        "nickname": "test",
+        "blocked": False,
+        "email_verified": True,
+        "phone_number": "+012345678",
+        "phone_verified": True,
+        "verify_phone_number": True,
+        "verify_password": True,
+    }
+    headers = {"Authorization": f"Bearer {auth0_secrets['access_token']}"}
+    users_response = requests.post(
+        url=f"{base_url}/dbconnections/signup", json=body, headers=headers
+    )
+    user = users_response.json()
+    assert 200 == users_response.status_code
+    error_message = (
+        f"User with email {auth0_erasure_identity_email} could not be added to auth0"
+    )
+    poll_for_existence(
+        _user_exists,
+        (auth0_erasure_identity_email, auth0_secrets),
+        error_message=error_message,
+    )
+    yield user
+
+    # Deleting user after verifying update request
+    headers = {
+        "Authorization": f"Bearer {auth0_secrets['access_token']}",
+    }
+
+    user_delete_response = requests.delete(
+        url=f"{base_url}/api/v2/users/auth0|{auth0_erasure_identity_email}",
```

Comment:
Is this delete URL correct? It looks like there might be a typo because of the `|` character.

Reply:
This worked fine as we needed to add /api/v2/users/<auth|id> but I have updated code not to contain '|' character in url.